Multisig/Features/Security/KeychainItem.swift

```diff
@@ -31,23 +55,23 @@ enum KeychainItem {
                 kSecReturnData: true,
             ]
 
-        case let .enclaveKey(tag, password, _):
+        case let .enclaveKey(tag, service, password, _):
             result = [
                 kSecClass: kSecClassKey,
                 kSecAttrKeyClass: kSecAttrKeyClassPrivate,
-                kSecAttrApplicationTag: tag.data(using: .utf8)!,
+                kSecAttrApplicationTag: (tag + ":" + service).data(using: .utf8)!,
                 kSecAttrKeyType: kSecAttrKeyTypeECSECPrimeRandom,
                 kSecReturnRef: true
             ]
 
             if let context = LAContext(password: password) {
                 result[kSecUseAuthenticationContext] = context
             }
-        case let .ecPubKey(tag, _):
+        case let .ecPubKey(tag, service, _):
             result = [
                 kSecClass: kSecClassKey,
                 kSecAttrKeyClass: kSecAttrKeyClassPublic,
-                kSecAttrApplicationTag: tag.data(using: .utf8)!,
+                kSecAttrApplicationTag: (tag + ":" + service).data(using: .utf8)!,
```

Comment:
could we extract this string pattern (tag + ":" + service) to a constant and reuse it?

Reply:
I can extract the whole thing to a function and/or the delimiter (":") to a constant. But `tag` and `service` are local variables here.